aws/ec2/resources.py

```diff
@@ -52,6 +52,36 @@ def ec2_ebs_volumes():
     )
 
 
+def ec2_ebs_snapshots():
+    "http://botocore.readthedocs.io/en/latest/reference/services/ec2.html#EC2.Client.describe_snapshots"
+    return (
+        botocore_client.get("ec2", "describe_snapshots", [], {"OwnerIds": ["self"]})
```

Comment:
Why are we restricting to "self" here? How are we guaranteed to iterate over all possible ownerid's? The docs suggest "self" filters out a lot of cases. The amazon owned ones we would want to exclude, but how do we know we aren't excluding volumes we care about?

It's not clear to me if ownerids are distinct from account ids, so I'm assuming:
- account ids are a subset of possible owner ids,
- the entire pytest aws suite is iterated over by account ids

While the code is clear, understanding what is the scope of testing is not. I'm not sure if that's solvable, or comprehension requires a deep mental model of all AWS objects.

Reply:
owner id == AWS account ID. In this test, we only care about EBS snapshots from "our" EBS volumes. "Our" in this case meaning the account that is being tested.